#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_qtdemo_destdir.dpatch by Fathi Boudra <fboudra@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix qtdemo destdir.

@DPATCH@
diff -urNad qt4-x11-4.3.0~/demos/qtdemo/qtdemo.pro qt4-x11-4.3.0/demos/qtdemo/qtdemo.pro
--- qt4-x11-4.3.0~/demos/qtdemo/qtdemo.pro	2007-05-25 15:24:38.000000000 +0200
+++ qt4-x11-4.3.0/demos/qtdemo/qtdemo.pro	2007-06-30 21:12:45.000000000 +0200
@@ -1,6 +1,6 @@
 CONFIG += assistant x11inc
 TARGET = qtdemo
-DESTDIR = $$QT_BUILD_TREE/bin
+DESTDIR = bin
 OBJECTS_DIR = .obj
 MOC_DIR = .moc
 INSTALLS += target sources
diff -urNad qt4-x11-4.3.0~/demos/arthurplugin/arthurplugin.pro qt4-x11-4.3.0/demos/arthurplugin/arthurplugin.pro
--- qt4-x11-4.3.0~/demos/arthurplugin/arthurplugin.pro	2007-05-25 15:24:45.000000000 +0200
+++ qt4-x11-4.3.0/demos/arthurplugin/arthurplugin.pro	2007-06-30 21:18:35.000000000 +0200
@@ -3,7 +3,7 @@
 
 CONFIG      += designer plugin debug_and_release
 TEMPLATE    = lib
-QTDIR_build:DESTDIR     = $$QT_BUILD_TREE/plugins/designer
+QTDIR_build:DESTDIR     = plugins/designer
 
 contains(QT_CONFIG, opengl) {
 	DEFINES += QT_OPENGL_SUPPORT
